CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Validate `NodeIndex` depth ([#482](https://github.com/0xMiden/crypto/pull/482)).
 - [BREAKING] Rename `ValuePath` to `MerkleProof` ([#483](https://github.com/0xMiden/crypto/pull/483)).
 - Added an implementation of Keccak256 hash function ([#487](https://github.com/0xMiden/crypto/pull/487)).
+- Added `{Smt,PartialSmt}::inner_node_indices` to make inner nodes accessible ([#494](https://github.com/0xMiden/crypto/pull/494)).
```

Comment:
This should go into the v0.17.0 section. Alternatively, since this is a non-breaking change, we could create a v0.16.1 release for this (and then we should change the base of the PR to `main`).

Reply:
v0.16.1 I could release today/tomorrow. v0.17.0 probably would be in a few weeks.